client/blocks/login/index.jsx

```diff
@@ -26,6 +27,7 @@ class Login extends Component {
 
 	state = {
 		rememberMe: false,
+		notice: null
```

Comment:
I don't think we need to use component state here, as I believe we can pull this information from the global store. I'll investigate.